RecoLocalTracker/Phase2TrackerRecHits/python/Phase2TrackerRecHits_cfi.py

```diff
@@ -3,7 +3,5 @@
 # RecHits options
 siPhase2RecHits = cms.EDProducer("Phase2TrackerRecHits",
   src = cms.InputTag("siPhase2Clusters"),
-  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPEDummy")
+  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPETrivial")
```

Comment:
Right, I could change this right away to Phase2StripCPEGeometric, if that's what you mean.